### DISCUSSION TITLE
TF-IDF matrix and differential accessible analysis

### DISCUSSION THREAD

**Discussion by qingwang1** [GreenleafLab/ArchR#1013] [Questions / Documentation]
Hi,
I know there are already question posted about where to find TF-IDF matrix, and your answer is it's not stored anywhere, and what I get from getMatrixFromProject() is raw count, I am just wondering is there any way to get the TF-IDF normalized count? if not, what it be ok to change it to seurat obj and use signac to TF-IDF normalized? Also if the TF-IDF matrix is not stored, what's used for the pairwise differential accessible analysis using getMarkerFeatures(), is that the raw count? Many thanks!

**Comment by rcorces** [comment]
Are you saying that Signac computes differential accessibility based on the TF-IDF normalized data? I dont believe that is the case.
TF-IDF and SVD are only used in ArchR for dimensionality reduction. Otherwise, counts are normalized based on Reads in TSS regions by default.

**Reply by qingwang1** [reply]
Thanks for your reply! I believe that Signac uses the 'data' slot which is the TF-IDF normlized count? for the differential accessibility. If we want to do a sanity check on the peak count, how should we get the counts normalized based on Reads in TSS region from the archr object then? Thanks!

**Reply by rcorces** [reply]
ArchR really is not designed to return that normalized matrix at the single-cell level. That is not a straightforward operation.
